.github/workflows/test.yaml

```
@@ -87,8 +112,16 @@ jobs:
         echo "Configuring Yaks"
         yaks install
 
+    - name: Run Tests
+      run: |
+        # Configure install options
+        export KAMEL_INSTALL_BUILD_PUBLISH_STRATEGY=Spectrum
+        export KAMEL_INSTALL_REGISTRY=$KIND_REGISTRY
+        export KAMEL_INSTALL_REGISTRY_INSECURE=true
+
         echo "Running tests"
-        yaks test ./test
+        yaks run --logs --wait ./test
```

Does not seem to wait...
```suggestion
        yaks test ./test
```

---

yeah, it was an attempt to see if for some reason the cli was not working as expected
btw, yaks test does not seems to exists as a sub-command is that correct ?